Gruntfile.js

```diff
@@ -125,7 +125,7 @@ module.exports = function(grunt) {
     usebanner: {
       dist: {
         options: {
-          position: 'top',
+         position: 'top',
```

Comment:
Missing space, please revert this change

Reply:
okay